src/ansys/fluent/core/services/datamodel_se.py

```diff
@@ -16,8 +16,8 @@
 
 
 class Attribute(Enum):
-    """Class containing the standard names of data model attributes associated
-    with the datamodel service."""
+    """Contains the standard names of the data model attributes associated with
+    the datamodel service."""
```

Comment:
Summary of docstrings should be on one line.
```suggestion
    """Contains the standard names of the data model attributes associated with the datamodel service."""
```